uw-frame-components/portal/misc/partials/launch-button.html

```diff
@@ -0,0 +1,5 @@
+<a class="launch-app-button ng-scope"
+   target="{{ target }}"
+   ng-href="{{ href }}">
```

Comment:
Could even have a sensible default like [the guidance](http://uw-madison-doit.github.io/angularjs-portal/latest/#/md/widget-launch-button) suggests.